core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -188,7 +188,7 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging with RawlsInstrum
       }
     }
 
-    def abortActiveWorkflows(submissionId: UUID) = {
+    def abortActiveWorkflows(submissionId: UUID): Future[Seq[Future[(Option[String], Try[ExecutionServiceStatus])]]] = {
```

Comment:
hah. i was about to say "but then you lose the `externalId`s in the return type!", and then realised we're throwing away the result anyway. will incorporate, thanks.